src/main/scala/com/signalcollect/triplerush/TripleRush.scala

```diff
@@ -148,10 +151,13 @@ case class TripleRush(
    * The placement hint should ensure that this gets processed on node 0, because the dictionary resides on that node.
    * If you get a serialization error for the dictionary, it is probably due to a problematic placement hint.
    */
-  def load(filePath: String, placementHint: Option[Long] = Some(OperationIds.embedInLong(1))): Unit = {
-    graph.loadGraph(new FileLoader(filePath, dictionary), placementHint)
+  def load(filePath: String, placementHint: Option[Long] = Some(OperationIds.embedInLong(1)), lang: Option[Lang] = None): Unit = {
+    graph.loadGraph(new DataLoader(Left(filePath), dictionary, lang), placementHint)
   }
 
+  def loadStream(inputStream: InputStream, placementHint: Option[Long] = Some(OperationIds.embedInLong(1)), lang: Option[Lang] = None): Unit = {
```

Comment:
ok..how about a default TURTLE? or should we just force a language for correctness?